src/Geometry-Tests/GEllipseTest.class.st

```diff
@@ -29,6 +35,30 @@ GEllipseTest >> testBoundaryContains [
 	self deny: (ellipse boundaryContains: 3, 0.13).
 ]
 
+{ #category : #tests }
+GEllipseTest >> testEmpty [
```

Comment:
Oh, so the code managed ellispes without vertexes.

I think we are in the same case of the rectangle, those are not ellispses. 

`An ellipse is a curve that is the locus of all points in the plane the sum of whose distances r_1 and r_2 from two fixed points F_1 and F_2 (the foci) separated by a distance of 2c is a given positive constant 2a (Hilbert and Cohn-Vossen 1999, p. 2). `

An ellipse needs a curve, which is not possible when there are no vertexes :(

Reply:
For roassal when shapes were empty I will use points

Reply:
Maybe the constructors of GRectangle and GEllipse can return a point instead of an error